shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -771,7 +771,7 @@ produced ::
   Core.TxBody era ->
   Core.Value era
 produced pp stakePools tx =
-  balance (txouts tx)
+  balance @era (txouts tx)
```

Comment:
It would be nice if the API requires less of this kind of thing instead of more of it.

Reply:
Many of these type applications are now gone. by making HashAnnotated a ternary type class.